### ISSUE TITLE
Inquiry about the dataset

### ISSUE THREAD

**Issue by fyl01** [YangLiangwei/FeSoG#3]
Regarding the filmtrust_FedMF.pkl dataset, I printed it out for inspection and found that whether it is the training set, validation set or test set, only the first 740 users have interaction records and the rest do not. Is this a special treatment or is the real data like this?

I would also like to ask how to preprocess the data to obtain such a pkl file.

Thank you.
